### PR TITLE
Separates example from library source

### PR DESCRIPTION
**Pull request by mayavera**
> ⚠️ blocked by #40 _Removed unused deps, upgraded deps to latest_

# Changelog
- moved example code from `src` to `examples/using-module`
- created directory for `examples/using-lib`
- cleaned up `examples/using-lib` webpack config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,7 +72,11 @@
     "eslint-plugin-standard": "^4.0.0",
     "html-loader": "0.5.5",
     "html-webpack-plugin": "3.2.0",
+    "mini-css-extract-plugin": "^0.5.0",
+    "node-sass": "^4.11.0",
+    "normalize-scss": "^7.0.1",
```
Comment (josephspens) [inline package.json]: Let's not start adding unnecessary packages, we can normalize css by including a CDN embedded on the page.

File: package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "npm": ">=8"
   },
   "scripts": {
-    "build": "webpack --mode=production --color --progress",
-    "start": "webpack-dev-server --config webpack.config.dev.babel.js --hot --progress",
+    "build": "webpack --mode=production --progress",
+    "start": "./node_modules/.bin/webpack-dev-server --config examples/using-module/webpack.config.babel.js --context examples/using-module --open",
```
Comment (josephspens) [inline package.json]: Why do we need to specify `node_modules` here? Doesn't NPM automatically look there?

File: package.json (hunk 1 above)
Comment (josephspens) [reply package.json]: With this comment I'm referring to all 5 of the packages just added.

Comment (mayavera) [reply package.json]: What are you talking about?

File: package.json (hunk 2 above)
Comment (mayavera) [reply package.json]: nice catch!

File: package.json (hunk 1 above)
Comment (mayavera) [reply package.json]: Why would we include an external dependency to test the project locally?

Comment (josephspens) [reply package.json]: I'm saying we don't need sass, so let's not add it

**Pull request merged**